### ISSUE TITLE
Question about manual testing file

### ISSUE THREAD

**Issue by Adoby7** [nus-cs2103-AY1718S1/forum#209]
Hi, prof! @damithc 
According to the Project v1.5 admin, we need to add SampleData.xml file for manual testing. However, our app needs two .xml files to store the data. Could we add SampleData1.xml and SampleData2.xml instead?

Thank you.

**Comment by tshradheya** [comment]
Another query that I had related to this was, whether the testers will replace the addressbook.xml with this sampleData.xml?

It might not work correctly cause thr name of files are different

**Comment by damithc** [comment]
@Adoby7 Yes, you can have two files. You should mention the need for two files in the test script file.
@tshradheya In your test script file you should explain exactly what the tester needs to do. If the file needs to be renamed, you should mention it there.

**Comment by yunpengn** [comment]
Hi, prof @damithc !

Is that true only one person of the whole team needs to upload `SampleData.xml` and `TestScript.pdf` to IVLE? Or everyone needs to do so?

**Comment by damithc** [comment]
> Is that true only one person of the whole team needs to upload SampleData.xml and TestScript.pdf to IVLE? Or everyone needs to do so?

It's a team submission. Yes, just one person should upload it on behalf of the team.

**Comment by erik0704** [comment]
Hi prof @damithc 

If I understand correct, since the test uses jar file, that would mean the app must have an import data features ?

**Comment by damithc** [comment]
> If I understand correct, since the test uses jar file, that would mean the app must have an import data features ?

Not necessarily. For example, you should be able to load  sample data by copying over content from the sample data file to the actual data file, right?
